doc/faq.rst

```diff
@@ -26,6 +26,21 @@ See :ref:`contributing`. Before wanting to add a new algorithm, which is
 usually a major and lengthy undertaking, it is recommended to start with :ref:`known
 issues <easy_issues>`.
 
+What's the best way to ask questions about scikit-learn?
+--------------------------------------------------------------
+The quickest way to get replies to your questions is posting on 
+`Stack Overflow <http://stackoverflow.com>`_ or 
+`Cross Validated <http://stats.stackexchange.com>`_ 
+ with the "scikit-learn" or "machine learning" tags. These sites may already have an answer
+ for your question and provide a convenient search interface. You can also post questions
+ on the mailing list, though that often has longer turn-around times.
+ Please don't ask general machine learning questions on the mailing list.
+
+For bug reports or feature requests, please make use of the
+`issue tracker on Github <https://github.com/scikit-learn/scikit-learn/issues>`_,
+the `mailing list <https://mail.python.org/mailman/listinfo/scikit-learn>`_, or
+the 'Scikit-learn community on Gitter <https://gitter.im/scikit-learn/scikit-learn>'_.
```

Comment:
i think you wanted backticks here, not single quotes. also this might look better with the S in "Scikit" lowercased